src/csharp/Grpc.Tools.TestData/testAtInPath/msbuildtest.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!--<Target Name="Build">
+    <Message Text="Building msbuildintro" />
+    <MSBuild Projects="msbuildintro.csproj" Targets="Build" />
+  </Target>-->
+
+    <Import Project="$(GrpcToolsBuildDir)\_grpc\_Grpc.Tools.props"/>
+    <Import Project="$(GrpcToolsBuildDir)\_protobuf\Google.Protobuf.Tools.props"/>
+
+  <PropertyGroup>
+    <Protobuf_Generator>CSharp</Protobuf_Generator>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Protobuf_IntermediatePath>obj/Debug/netstandard2.0</Protobuf_IntermediatePath>
```

Comment:
rather than requiring `<Protobuf_IntermediatePath>` in each of our test cases, should the TryRunMsBuild inject this msbuild property?
I think it would be good for maintainability and readability if the test .csproj files looked almost exactly like "real world" .csproj files (i.e. there wouldn't be any non-standard properties in each of the files).

Another idea would be to include all the test-specific settings in a  props/targets file and only included that file:

(e.g. MsBuildIntegrationTestEnvironment.props/.targets"). This .props .targets file could also be injected automatically into all test-case .csproj files via Directory.Build.props and Directory.Build.targets.